### PR TITLE
[36] Add getFiles method

### PR DESCRIPTION
**Pull request by kevinwaelkens**
#36 

This PR adds the getFiles method. It returns all the files' information of your onesky project.

I blatantly copied the `getFile` method and tests for the `getFiles` method because it's nearly identical. I only got rid of some params which are of no use for this request.

### CODE REVIEW

File: lib/getFiles.js [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+}
+
+/**
+ * @param  {Object} options
+ * @return {String}
+ * @private
+ */
+function _getLink (options) {
+  return apiAddress + '/1/projects/' + options.projectId + '/files?' + queryString.stringify({
+    api_key: options.apiKey,
```
Comment (maciekbb) [inline lib/getFiles.js]: Not sure if `api_key` should be sent with GET request

Comment (kevinwaelkens) [reply lib/getFiles.js]: It's necessary to authenticate your request  - https://github.com/onesky/api-documentation-platform#parameters

**Comment by kevinwaelkens** [comment]
Thanks for the quick replies guys 🙏

File: package.json [hunk 2]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "@brainly/onesky-utils",
-  "version": "1.3.2",
+  "version": "1.3.3",
```
Comment (SebastianCzoch) [inline package.json]: please set version to `1.4.0` it's not security/bug release, just a new feature added

**Comment by SebastianCzoch** [comment]
@kevinwaelkens Thank you for your contribution, new version was relased

**Pull request merged**